docs/examples/dali_plugin/custom_copy_client.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python
```

Comment:
I fell like this example doesn't need a client script

Reply:
I added a client to actually test, that it works. When we'll have unified client for tests, we could omit this

Reply:
So I would add a client when we actually test it (and the client will probably go to a corresponding L0_ directory and not here)

Reply:
I think it would be nice to separate testing files from example files not to confuse readers

Reply:
Done